frontend/src/features/mapSideBar/SidebarContents/subforms/AssociatedLandReviewPage.tsx

```diff
@@ -132,7 +132,7 @@ export const AssociatedLandReviewPage: React.FC<any> = (props: IReviewProps) =>
                         className="edit"
                         onClick={() =>
                           setEditInfo({
-                            ...defaultEditValues,
```

Comment:
This was causing the other sections to change when an unrelated edit icon was clicked. I think I like it better if only the clicked section changes.